src/RA_AchievementSet.cpp

```diff
@@ -381,7 +379,7 @@ BOOL AchievementSet::FetchFromWebBlocking(GameID nGameID)
         doc.HasMember("PatchData"))
     {
         const Value& PatchData = doc["PatchData"];
-        SetCurrentDirectory(NativeStr(g_sHomeDir).c_str());
+        ChangeToHomeDirectory();
```

Comment:
It's called everywhere, for now it's discardable but we'll put some checks in later.